brave/brave-core
# [Lit]: Migrate br-toolbar from Polymer to Lit

Resolves https://github.com/brave/brave-browser/issues/41964



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Navigate to `chrome://settings`
2. Toolbar should show up
3. Buttons / icons should be clickable
4. Search should work
5. Pressing escape to close search should work
